flow/accounts/create/create.go

```diff
@@ -70,7 +73,28 @@ var Cmd = &cobra.Command{
 			}
 		}
 
-		tx := templates.CreateAccount(accountKeys, nil, signerAccount.Address)
+		var contracts []templates.Contract
+
+		if len(conf.Contract) > 0 {
+			contractFlagContent := strings.Split(conf.Contract, ":")
```

Comment:
```suggestion
			contractFlagContent := strings.SplitN(conf.Contract, ":", 2)
```

Reply:
I didn't SplitN so that i could use the length as an extra check, but `:` in filename/path would probably just make it error on read later, which is probably okay.